index.html

```diff
@@ -20,10 +20,16 @@
         <li class="cellular"><img src="assets/Bars/Status Bars/Cellular Connectionmobile-top.png"></li>
         <li class="wifi"><img src="assets/Wifiwifi.png"></li>
         <li class="battery"><img src="assets/Bars/Status Bars/Batterymobile-top.png"></li>
```

Comment:
- [ ] I would suggest you kindly remove this part from your project as it is not part of the project design and requirement, It was actually there in the Figma design to show you that the design is on a mobile view.
Kindly remove it 👍 👍